extensions/amp-sidebar/1.0/component.js

```diff
@@ -191,44 +191,49 @@ export function SidebarToolbar({
   ...rest
 }) {
   const ref = useRef();
-  const [matches, setMatches] = useState(false);
-  const [targetEl, setTargetEl] = useState(null);
+  const [target, setTarget] = useState();
 
   useEffect(() => {
```

Comment:
Separate effect for finding the `target` element.  This way if only the `mediaQuery` is updated, we do not need to recalculate the `target` element.